provision.yml

```diff
@@ -9,7 +9,7 @@
     - role: nginx
       vhosts:
         - name: staging.stockholmsaltspa.se
-          local_port: 8080
+          local_port: 80
```

Comment:
Nawh, this can't be. Port 80 and 443 is needed by Nginx. You can only bind one application to a specific port at a time. Make your WordPress Docker image that you're proxying to run on port 8080 instead. 👍 ⚡️